examples/pre-commit.nix

```diff
@@ -1,15 +0,0 @@
-{ inputs, config, ... }: {
-  imports = [
-    inputs.pre-commit-hooks.flakeModule
-    config.flake-lang.rustMonorepoPreCommit
-    config.flake-lang.denoPreCommit
```

Comment:
Just a remark -- I don't think we can do this i.e., import a value set in the `config`. In Slack, I've cooked up a minimal example to demonstrate.

Reply:
That makes a lot of sense!